src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
It wouldn't catch `window.setTimmeout` for example, our previous use of `globals` in the `.jshintrc` prevented such spelling errors. It would seem that even some basic unit tests would prevent those kinds of issues though.